.github/workflows/continuous-integration.yml

```diff
@@ -424,7 +424,9 @@ jobs:
           set -x
 
           image_name="$DOCKER_IMAGE_NAME$suffix"
-          img_version="$image_name:$VERSION$ci_run"
+          img_version_raw="$image_name:$VERSION$ci_run"
+          # Replace unsupported character `+`
+          img_version="${img_version_raw//[+]/__}"
```

Comment:
@tgrospic, you used another way than in [this PR](https://github.com/rchain/rchain/pull/3550/files). What is the benefit of the current solution?

Reply:
It should produce the same image tag. Here is when it's pushed on DockerHub and in previous PR is when image is built locally.

Reply:
So if we want to build image locally from sbt and then push it to DockerHub, we should manually rename image?

Reply:
With every push to DockerHub we need to create image name and tag which correspond to specific user account, so it's not the same as locally build by `docker:publishLocal` sbt which is like this:
https://github.com/rchain/rchain/runs/4454568399?check_suite_focus=true#step:7:444
```
[info] Successfully tagged coop.rchain/rnode:0.12.4__rholang-1.1-rc1
[info] Successfully tagged coop.rchain/rnode:latest
```
So here we need to tag it again by using any of the locally created tags (created by `build_docker_image` job) and create a new tag which we want to push. The source to get image tag here is Git tag so that's why fixing the `+` is done again here.

Maybe another solution can be to query local images and reuse image name and tag but this would involve more changes right now.